assessments/homework.sql

```diff
@@ -93,7 +94,8 @@ INSERT INTO question_score_logs
 UPDATE instance_questions AS iq
 SET
     points_in_grading = least(iq.points + iq.current_value, aq.max_points) - iq.points,
-    score_perc_in_grading = least(iq.points + iq.current_value, aq.max_points) / aq.max_points * 100 - iq.score_perc
+    score_perc_in_grading = least(iq.points + iq.current_value, aq.max_points)
```

Comment:
I honestly don't know. If it should be points_in_grading, then I'll change it over. I'm not familiar enough with the overall workings to know the difference.